USING.md

```diff
@@ -27,18 +27,29 @@ on the computer and that you can run Docker on it.
 When Docker has been correctly installed, no more installation is needed to run
 `zonemaster-cli`. Just follow the examples below.
 
+There is a limitation in Docker regarding IPv6. Unless IPv6 has been enabled in
+the Docker daemon, there is no support for IPv6. To avoid meaningless errors,
+use `--no-ipv6` if there is no IPv6 support. Also see "[Enable IPv6 support]".
+
 ### Local installation
 
 To have an local installation of Zonemaster-CLI follow the
 [installation instruction]. When installed, the examples below can be followed.
 
+If the network has no IPv6 support (common in home networks) then turn off IPv6.
+Use `--no-ipv6` to avoid meaningless errors if there is no IPv6 support.
+
 
 ## Invoking the command line tool
 
 The most basic use of the `zonemaster-cli` command is to just test a domain, e.g.
 "zonemaster.net".
 
-With Docker:
+Using Docker:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6
+```
+or
 ```sh
 docker run -t --rm zonemaster/cli zonemaster.net
 ```
```

Comment:
Since I understand Docker is not configured with IPv6 by default, I think we can leave only one command that most users will use (the one with `--no-ipv6`) and remove the second one (letting people remove the option if needed).

Reply:
Can we leave as-is for now, and tweak for next release?